src/utils.ts

```diff
@@ -55,6 +55,16 @@ export function findMarkerMatch(text: string, markers: MarkerData[]): MarkerData
 	return null;
 }
 
+export function isMarkerMatchAt(key: string, markers: MarkerData[], index: number): boolean {
+	for (const marker of markers) {
+		if (key === marker.text[index]) {
```

Comment:
Maybe we should use `startsWith` instead of this.

So we use `if (marker.text.startsWith(key)) {`. This would remove `index` from the arguments.

Reply:
Great idea! But just to clarify, you meant `if (marker.text.startsWith(input)) {`, right?

Reply:
Yeah, I think you pass `input` from the plugin into the `key` field, when you call the `isMarkerMatchAt` function.